src/test/ui/llvm-asm/issue-37508.rs

```diff
@@ -0,0 +1,35 @@
+// compile-flags: -Ccode-model=large --crate-type lib
```

Comment:
Note: I haven't taken a closer look but this may contain a failure similar to https://github.com/rust-lang/rust/pull/84508#issuecomment-828908291.